devops/staging/values.yaml

```diff
@@ -6,7 +6,8 @@ iam-cache-server-helm:
   replicaCount: 3
 
   image:
-    repository: "887755037386.dkr.ecr.us-west-2.amazonaws.com/ics-api"
+    repository: "ghcr.io/energywebfoundation/ssi-hub"
+    tag: v2.13.0
```

Comment:
With this change, the staging environment will use this ssi-hub release, instead of building an image from the `master` branch